example/src/components/ResponsiveColors.scss

```diff
@@ -0,0 +1,33 @@
+.responsive-colors {
+  position: relative;
+  color: white;
+  background-color: brown;
+  max-width: 100%;
+  margin-top: 20px;
+  padding: 10px;
+  box-sizing: border-box;
+
+  &--width-small {
+    background-color: red;
+  }
+
+  &--width-medium {
+    background-color: orange;
+  }
+
+  &--width-large {
+    background-color: green;
+  }
+
+  &--height-short {
+    border: 5px solid purple;
+  }
+
+  &--height-medium {
+    border: 5px double yellow;
+  }
+
+  &--height-long {
+    border: 5px dotted hotpink;
+  }
+}
```

Comment:
Is this in the right folder? I expected to see it in /scss instead of /components